CHANGELOG.md

```diff
@@ -1,3 +1,126 @@
+### citus v11.1.0 (September 15, 2022) ###
+
+* Adds support for PostgreSQL 15beta4
+
+* Adds a rebalancer that uses background tasks for its execution
+
+* Adds `create_distributed_table_concurrently()` UDF to distribute tables
+  without blocking writes
+
+* Adds `citus_split_shard_by_split_points()` UDF that allows
+  splitting a shard to specified set of nodes without blocking writes
+  and based on given split points
+
+* Adds support for non-blocking tenant isolation
+
+* Adds support for isolation tenants that use partitioned tables
+  or columnar tables
+
+* Drops support for isolation tenants that use replicated tables
+
+* Separates columnar table access method into a separate logical extension
+
+* Adds support for logical replication in `replicate_reference_tables()`
+
+* Improves performance of blocking shard moves
+
+* Improves non-blocking shard moves with a faster custom copy logic
+
+* Adds the GUC `allow_unsafe_constraints` to allow constraints without
+  distribution column
+
+* Introduces GUC `citus.skip_constraint_validation`
+
+* Replaces `citus.hide_shards_from_app_name_prefixes` GUC with
+  `citus.show_shards_for_app_name_prefixes`
+
+* Introduces `citus_locks` view
+
+* Improves `citus_tables` view by showing local tables added to metadata
+
+* Improves columnar table access method by moving old catalog tables into
+  an internal schema and introduces more secure & informative views based
+  on them
+
+* Removes `do_repair` option from `citus_copy_shard_placement()`
```

Comment:
As similar to earlier change log entries, can't we group `Adds` and `Removes` back to back, instead of scattered around?

For example, `Drops support for isolation tenants that use replicated tables` should also be close to here (and if we have ay others).

And, also all `removes/drops` could come after all `adds` and before `fixes`